src/main/java/world/bentobox/bentobox/api/commands/CompositeCommand.java

```diff
@@ -821,4 +826,33 @@ protected boolean checkCooldown(User user, UUID targetUUID) {
         user.sendMessage("general.errors.you-must-wait", TextVariables.NUMBER, String.valueOf(timeToGo));
         return true;
     }
+
+    /**
+     * @return the onlyConsole
+     */
+    public boolean isOnlyConsole() {
+        return onlyConsole;
+    }
+
+    /**
+     * This command is only for console use
+     */
+    public void setOnlyConsole() {
+        this.onlyConsole = true;
+    }
+
+    /**
+     * @return the configurableRankCommand
+     */
+    public boolean isConfigurableRankCommand() {
+        return configurableRankCommand;
+    }
+
+    /**
+     * This command can be configured for use by different ranks
+     */
+    public void setConfigurableRankCommand() {
```

Comment:
I think it should be renamed to `useConfigurableRank`. And this method should have a boolean parameter.

Reply:
This just comes from the variable (configurableRankCommand), which is true if the command is a configurable rank command. CongifurableRank would have a different meaning in my mind. No need for a boolean parameter as it's only used if it is going to be set to true.